spec/fluent/plugin/out_logtail_spec.rb

```diff
@@ -8,6 +8,13 @@
     }
   end
 
+  let(:cloud_config) do
+    %{
+      source_token  abcd1234
+      ingesting_host s1234.g1.betterstackdata.com
```

Comment:
@lazandrei19 if you're stubbing this anyway, let's use the production cluster (`s1234.eu-nbg-2.betterstackdata.com`)?